### ISSUE TITLE
"Game could not be started" on  Star Wars Lethal Allience

### ISSUE THREAD

**Issue by Milach330** [PSP-Archive/ARK-4#384]
### PSP/Vita Model

PSP-N1000 (PSP Go)

### ARK Version

ARK 4.20.69 r9

### Homebrew/GAME (if applicable)

ULUS-10188

### Plugins ( if applicable )

game, ef0:/SEPLUGINS/nploader.prx, on
vsh, ef0:/SEPLUGINS/nploader.prx, on
pops, ef0:/SEPLUGINS/nploader.prx, on

### Steps to reproduce

1. Boot up PSP
2. Run Star Wars Lethal Allience
3. After going to title screen PSP goes back to XMB with error "The game could not be started. (80010016)"

### USB Charge

Disabled

### Overclock

Disabled

### Power Save

Disabled

### Balanced Energy Mode

Disabled

### Autoboot Launcher

Off

### Disable PSP Go Pause

Off

### Unlock Extra Memory

Disabled

### Memory Stick Speedup

Disabled

### Inferno Cache

Off

### Skip Sony Logos

Off

### Hide PIC0 and PIC1

Off

### Hide MAC Address

Off

### Hide DLC

Off

### Turn off LEDs

Disabled

### Verification of form.

- [X] I agree that I properly filled out the items listed above.

**Comment by krazynez** [comment]
Try booting holding Start and see if it continues onwards. It seems to work fine for me with no plugins.

**Comment by danberm2003** [comment]
was having the same issue you need to go into vhs menu by pressing select+start and change the umd iso mode to M33 Driver

**Comment by krazynez** [comment]
> was having the same issue you need to go into vsh menu by pressing select+start and change the umd iso mode to M33 Driver

Thats not a thing on ARK

**Comment by JoseAaronLopezGarcia** [comment]
This sounds like the game wasn't copied correctly or the memory card is corrupted. Try copying the game again.

**Comment by Milach330** [comment]
I copied a copy of game few time to be sure, also checked MD5 checksum and it was correctm also tried regenerating ISO and CSO by UMDGEN.  Also on the same device and file on PRO-C it's working without issues even on Inferno driver which is the only only one on ARK.

**Comment by krazynez** [comment]
Did you disable the plugins as I stated before? Those might be interfering.

**Comment by Milach330** [comment]
Yes, I tried to disable all plugins and it didn;t help. If it help I tired to run  it from internal memory (I don't have memory stick adapter yet) but on the same configuration  and UMD driver on PRO-C it works

**Comment by krazynez** [comment]
@JoseAaronLopezGarcia can you try it since you have a GO and I don't.

**Comment by krazynez** [comment]
Works fine on latest release. Tested on my GO (now that I have one)

**Comment by alan-morford** [comment]
I just wanted to say I'm on a fresh install of the latest ARK4 on my PSP Go and I also cannot load this game. I tried 3 different ISOs from different sources and they all have the same result.

**Comment by JoseAaronLopezGarcia** [comment]
Working fine on latest release.
